src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -173,4 +175,28 @@ protected function extractProperties(Request $request) : array
 
         return $request->query->all();
     }
+
+    /**
+     * @param array property
```

Comment:
In our use cases the exception would not be thrown, in case the user want to use the methods without a nested property he will have to catch the error or/and to test with `$this->isPropertyNested` so it's ok to add the exception for me.